llvm/include/llvm/Support/Path.h

```diff
@@ -48,6 +48,14 @@ constexpr bool is_style_posix(Style S) {
 /// Check if \p S uses Windows path rules.
 constexpr bool is_style_windows(Style S) { return !is_style_posix(S); }
 
+constexpr Style system_style() {
```

Comment:
The only place this is used is to check if it's posix.  Instead we should check `is_style_posix(Style::native)` which already does the right thing, and lets us avoid modifying Path.h.